aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2447,6 +2447,10 @@ Tensor movedim(const Tensor& self, IntArrayRef src, IntArrayRef dst) {
   TORCH_CHECK(all_unique(normalized_src), "movedim: repeated dim in `source` (", src, ")");
   TORCH_CHECK(all_unique(normalized_dst), "movedim: repeated dim in `destination` (", dst, ")");
 
+  // handle the case of scalar tensor as a no-op
+  if (self_dim == 0)
+    return self.view(self.sizes());
```

Comment:
nit: `self.alias()` would be simpler here.

Reply:
rigth, done!